app/helpers/movies_helper.rb

```diff
@@ -26,8 +26,62 @@ def trailer_button_text(movie)
     movie.trailer.present? ? 'Change trailer' : 'Add a trailer'
   end
 
-  def movie_stats(movie)
-    "#{movie.release_date.stamp("2001") if movie.release_date.present?} | #{movie.mpaa_rating} | ★ #{movie.vote_average} /10 | #{movie.runtime_display}"
+  def movie_stats_display(movie)
+    movie_stats = [
+      release_date_display(movie),
+      movie.mpaa_rating,
+      runtime_display(movie),
+      star_rating(movie)
+    ]
+    movie_stats.compact.join(" | ")
+  end
+
+  def movie_genres_display(movie)
+    output = raw(movie.genres.map do |genre|
+      link_to genre, genre_path(genre)
+    end.join(", "))
```

Comment:
sweet. i didn't know about `.prepend`

Reply:
Me neither! It was in my head because of Javascript and I just tried it.